easybuild/easyconfigs/m/matplotlib/matplotlib-2.2.4-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'matplotlib'
+version = '2.2.4'
```

Comment:
@SethosII Please remove this easyconfig, it's no longer required?

`eb --update-pr 8691 :matplotlib-2.2.4-foss-2018b-Python-3.6.6.eb` should do it (see https://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html#deleting-easyconfig-files-or-patches)